packages/forma-36-website/src/pages/guidelines/copy/grammar/word-choice/index.mdx

```diff
@@ -0,0 +1,46 @@
+# Word choice
+
+## Use contractions when possible
+
+To make a sentence more scannable, use contractions when possible. Note that in some cases the words can have more emphasis when they are not contracted (**Can not** has greater emphasis than **Can't**).
+
+![Use contractions when possible](/contractions.jpg)
+
+## Address users based on context (Your, My, We)
+
+Determine the appropriate pronoun based on the context. Focus on the user and the actions the user is taking whenever possible. Avoid mixing prounouns ("my" and "your”) in the same section— It can cause confusion to see both forms of addressing the user in the same context.
+
+### Second person (“you” or “your”)
+
+Use this style for most situations that require a pronoun (when the web app is speaking directly to the user).
+
+![Second person (you or your)](/2nd-person-pronoun.jpg)
+
+### First person (“my” or “I”)
+
+This may be used in a context that communicates the user's actions or ownership.
+
+![First person (I or my)](/first-person.jpg)
+
+### The pronouns “we” and “our”
+
+Overall, focus on the user rather than what you or your app is doing for the user. “We” should only be used in situations where the web app is taking an action for or on behalf of the user.
```

Comment:
Not sure how to reword this, but again, remove web app I think. Here's a (not very good) suggestion:

`“We” should only be used in situations where the web app is taking an action for or on behalf of the user.` -> `“We” should only be used in situations when an action is being performed for or on behalf of the user.`